packages/desktop-client/src/global-events.ts

```diff
@@ -19,58 +19,30 @@ import { listen } from 'loot-core/src/platform/client/fetch';
 import * as undo from 'loot-core/src/platform/client/undo';
 
 export function handleGlobalEvents(store: AppStore) {
-  listen('server-error', () => {
+  const unlistenServerError = listen('server-error', () => {
     store.dispatch(addGenericErrorNotification());
   });
 
-  listen('orphaned-payees', ({ orphanedIds, updatedPayeeIds }) => {
-    // Right now, it prompts to merge into the first payee
-    store.dispatch(
-      pushModal('merge-unused-payees', {
-        payeeIds: orphanedIds,
-        targetPayeeId: updatedPayeeIds[0],
-      }),
-    );
-  });
-
-  listen('schedules-offline', () => {
-    store.dispatch(pushModal('schedule-posts-offline-notification'));
-  });
-
-  // This is experimental: we sync data locally automatically when
-  // data changes from the backend
-  listen('sync-event', async event => {
```

Comment:
Moved this to the `shared-listeners.ts` file. That file has been updated to handle both `success` and `applied` events